napalm_junos/junos.py

```diff
@@ -615,16 +615,19 @@ def get_lldp_neighbors_detail(self, interface=''):
             log.error(rpcerr.message)
             return {}
         interfaces = lldp_table.get().keys()
-
-        old_junos = napalm_base.helpers.convert(
-            int, self.device.facts.get('version', '0.0').split('.')[0], 0) < 13
-
+        
+        # get lldp neighbor by interface rpc for EX Series, QFX Series, J Series
+        # and SRX Series is get-lldp-interface-neighbors-information, 
+        # and rpc for M, MX, and T Series is get-lldp-interface-neighbors
+        # ref1: https://apps.juniper.net/xmlapi/operTags.jsp  (Junos 13.1 and later)
+        # ref2: https://www.juniper.net/documentation/en_US/junos12.3/information-products/topic-collections/junos-xml-ref-oper/index.html  (Junos 12.3)
```

Comment:
For very long lines, pylama will complain even if they are just comments.
In cases like that, when the reason is a very long URL, you can add `# noqa` at the end of the line